sparkstreaming-kinesis/autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+zipkin.sparkstreaming.autoconfigure.kinesisStream.kinesis.ZipkinKinesisStreamFactoryAutoConfiguration
```

Comment:
the package name has mixed case which probably isn't right.

`zipkin.sparkstreaming.stream.kinesis`